cmd/keys.go

```diff
@@ -0,0 +1,16 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var keysCmd = &cobra.Command{
+	Use:              "keys",
+	Short:            "Manage keys in use by your factory fleet",
+	PersistentPreRun: assertLogin,
+}
+
+func init() {
+	rootCmd.AddCommand(keysCmd)
+	requireFactory(keysCmd)
```

Comment:
This command doesn't require the factory cli opts, so delete this line.

Reply:
removed